src/core/backbone_router/manager.cpp

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
It follows the same style of `backbone_router/local.cpp` and `backbone_router/leader.cpp`. The filename is short since it's already in the `backbone_router` folder.

Reply:
`backbone_router/leader.cpp` is what motivated #4754. As a result, I would prefer more specific file names.

Reply:
So, which do you prefer, `backbone_router_manager.cpp` or `bbr_manager.cpp`? @librasungirl @jwhui 
I personally prefer more concise names.

Reply:
I'm fine with `bbr_manager.cpp` especially given that this is in the `backbone_router` directory.

Reply:
renamed accordingly